analysis_options.yaml

```diff
@@ -98,43 +133,78 @@ linter:
     - prefer_const_literals_to_create_immutables
     - prefer_constructors_over_static_methods
     - prefer_contains
+#    - prefer_double_quotes # Conflicts with prefer_single_quotes
     - prefer_equal_for_default_values
     - prefer_expression_function_bodies
     - prefer_final_fields
+    - prefer_final_in_for_each
 #    - prefer_final_locals # Google Avoids. Does not reflect standard usage.
+#    - prefer_final_parameters # Conflicts with unnecessary_final
+    - prefer_for_elements_to_map_fromIterable
     - prefer_foreach
     - prefer_function_declarations_over_variables
     - prefer_generic_function_type_aliases
+    - prefer_if_elements_to_conditional_expressions
+    - prefer_if_null_operators
     - prefer_initializing_formals
+    - prefer_inlined_adds
+    - prefer_int_literals
     - prefer_interpolation_to_compose_strings
     - prefer_is_empty
     - prefer_is_not_empty
+    - prefer_is_not_operator
     - prefer_iterable_whereType
+    - prefer_mixin
+    - prefer_null_aware_method_calls
+    - prefer_null_aware_operators
     - prefer_single_quotes
+    - prefer_spread_collections
     - prefer_typing_uninitialized_variables
+    - provide_deprecation_message
     - public_member_api_docs
     - recursive_getters
+    - require_trailing_commas
+#    - sized_box_for_whitespace # Not a Flutter project
     - slash_for_doc_comments
+#    - sort_child_properties_last # Not a Flutter project
     - sort_constructors_first
     - sort_unnamed_constructors_first
+    - tighten_type_of_initializing_formals
     - type_annotate_public_apis
     - type_init_formals
     - unawaited_futures
+    - unnecessary_await_in_return
     - unnecessary_brace_in_string_interps
-#    - unnecessary_const # Not 1.24.3 compatible
+    - unnecessary_const
+    - unnecessary_final
```

Comment:
Enabling unnecessary_final conflicts with one of the new lints prefer_final_in_for_each.

Which should be enabled?